src/app/mainPageView.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
+import { Button } from "@mui/material";
+import Image from "next/image";
+import Link from "next/link";
+import { type Session } from "next-auth";
+
+type mainPageProps = {
+  session: Session | null;
+};
+
+// TO DO: complete the page (styles)
+export const MainPageView: React.FC<mainPageProps> = ({ session }) => {
+  return (
+    <main>
+      <div className="absolute left-0 top-0 z-0 h-full w-full bg-darkSecondary/[.5] "></div>
+      <Image
+        src="/main.jpg"
+        alt="main"
+        sizes="100%"
+        layout="fill"
+        objectFit="cover"
+        className="absolute left-0 top-0 z-[-1]"
+      />
+      <div className="absolute z-10 flex min-h-screen flex-col justify-center">
+        <h1 className="text-8xl text-white">Music App</h1>
+        <div className="mt-5 flex gap-3">
+          {!session && (
+            <Button
+              className="rounded-md border-2 border-textSecondary text-5xl  hover:border-mainRed hover:bg-mainRed hover:text-white"
+              variant="outlined"
+            >
+              <Link
+                href={"/login"}
```

Comment:
There's no need to use curly brackets here. But it's ok :)
```suggestion
                href="/login"
```